### PR TITLE
Uppsf 240 locaiton by cc

### PR DESCRIPTION
**Pull request by emildimitrov**
- Retrieve source by cc
- Enable returning of authorities which are not a concept

### CODE REVIEW

File: concordances/cypher.go [hunk 1]
```diff
@@ -36,22 +36,29 @@ func (pcw CypherDriver) ReadByConceptID(identifiers []string) (concordances Conc
 	var results []neoReadStruct
 	query := &neoism.CypherQuery{
 		Statement: `
-		MATCH (p:Concept)
+		MATCH (p:Thing)
```
Comment (MiroslavGatsanoga) [inline concordances/cypher.go]: Why we need to change from Concept to Thing in the query?

File: concordances/cypher.go [hunk 2]
```diff
@@ -109,6 +116,22 @@ func (pcw CypherDriver) ReadByAuthority(authority string, identifierValues []str
 		AND exists(p.prefUUID)
 		RETURN DISTINCT p.prefUUID AS canonicalUUID, labels(p) AS types, p.uuid as UUID, 'LEI' as authority, p.leiCode as authorityValue`,
 
+			Parameters: neoism.Props{
+				"authorityValue": identifierValues,
+			},
+			Result: &results,
+		}
+	} else if authorityProperty == "ISO-1366-1" {
+		//hacky way to retrieve prefUUID of the canonical
```
Comment (MiroslavGatsanoga) [inline concordances/cypher.go]: Is there a non hacky way of doing it?

**Comment by emildimitrov** [comment]
> Do we need the example.json file in the root folder, is it used by some tests?
No removed
> Another thing to note (not sure if it could cause problems) is that we use Debian based Docker image for Circle CI and Alpine based one for our deployments.
We use almost everywhere the circleCi image for testing etc. I don't think its a problem

**Comment by emildimitrov** [comment]
I'v updated the google doc, but editor must approve my suggestions
Link in readme is updated. Gsites updated

File: concordances/cypher.go (hunk 1 above)
Comment (emildimitrov) [reply concordances/cypher.go]: Yep in order to return DBPedia, Wikidata without prefLabel etc

File: concordances/cypher.go (hunk 2 above)
Comment (emildimitrov) [reply concordances/cypher.go]: Removed the comment it was misleading. We'r doing the same as with LEI

File: concordances/cypher.go [hunk 3]
```diff
@@ -36,22 +36,29 @@ func (pcw CypherDriver) ReadByConceptID(identifiers []string) (concordances Conc
...
 		WHERE exists(canonical.leiCode)
 		RETURN DISTINCT canonical.prefUUID AS canonicalUUID, labels(canonical) AS types, 'LEI' as authority, canonical.leiCode as authorityValue
 		UNION ALL
-		MATCH (p:Concept)
+		MATCH (p:Thing)
 		WHERE p.uuid in {identifiers}
 		MATCH (p)-[:EQUIVALENT_TO]->(canonical:Concept)
-		MATCH (canonical)<-[:EQUIVALENT_TO]-(leafNode:Concept)
+		WHERE exists(canonical.iso31661)
+		RETURN DISTINCT canonical.prefUUID AS canonicalUUID, labels(canonical) AS types, 'ISO-3166-1' as authority, canonical.iso13661 as authorityValue
```
Comment (MiroslavGatsanoga) [inline concordances/cypher.go]: Missed one (canonical.iso13661 should be canonical.iso31661)

File: concordances/cypher.go [hunk 4]
```diff
@@ -109,6 +116,20 @@ func (pcw CypherDriver) ReadByAuthority(authority string, identifierValues []str
...
+			Result: &results,
+		}
+	} else if authorityProperty == "ISO-3166-1" {
+		query = &neoism.CypherQuery{
+			Statement: `
+		MATCH (p:Concept)
+		WHERE exists(p.iso31661)
+		AND p.iso31661 IN {authorityValue}
+		MATCH (p)-[:EQUIVALENT_TO]->(canonical:Concept)
+		RETURN DISTINCT canonical.prefUUID AS canonicalUUID, labels(canonical) AS types, p.uuid as UUID, 'ISO-3166-1' as authority, p.iso13661 as authorityValue
```
Comment (MiroslavGatsanoga) [inline concordances/cypher.go]: And another one (p.iso13661 -> p.iso31661)

**Comment by emildimitrov** [comment]
> The test coverage dropped and I see that the new stuff is not being covered by tests, so there should be tests for the introduced feature.

There is separate JIRA ticket for adding the test, its because we need to update many parts of concepts-rw-neo4j and decided to do it after its done

**Pull request merged**